rag_experiment_accelerator/doc_loader/tests/test_custom_json_loader.py

```diff
@@ -25,11 +25,15 @@ def test_load_json_files():
             "keys_to_load": keys_to_load,
         },
     )
-
+    doc = list(docs[0].values())[0]
     assert (
-        list(docs[0].values())[0]
+        str(doc["content"])
         == "[{'content': 'This is the content for item 1.', 'title': 'Title TEST 1'}, {'content': 'This is the content for item 2.', 'title': 'Title 2'}, {'content': 'This is the content for item 3.', 'title': 'Title 3'}, {'content': 'This is the content for item 4.', 'title': 'Title 4'}, {'content': 'This is the content for item 5.', 'title': 'Title 5'}, {'content': 'This is the content for item 6.', 'title': 'Title 6'}]"
     )
+    assert (
```

Comment:
Wouldn't it be better in this case, and the change above, to validate the content of the doc, rather than attempt a string comparison?

Reply:
correct, much readable, fixed